xarray/tests/test_backends.py

```diff
@@ -399,6 +399,24 @@ def test_orthogonal_indexing(self):
             actual = on_disk.isel(**indexers)
             self.assertDatasetIdentical(expected, actual)
 
+    def test_vectorized_indexing(self):
+        # Make sure vectorized_indexing works or at leaset raises
+        # NotImplementedError
+        in_memory = create_test_data()
+        with self.roundtrip(in_memory) as on_disk:
+            indexers = {'dim1': DataArray([0, 2, 0], dims='a'),
+                        'dim2': DataArray([0, 2, 3], dims='a')}
+            expected = in_memory.isel(**indexers)
+            try:
+                actual = on_disk.isel(**indexers)
+                self.assertDatasetIdentical(expected, actual)
```

Comment:
FYI for the future `xarray.tests.assert_identical` is a better interface

Reply:
I adopted `assert_identical` only where I added. 
After the approval, I will change the other part of `test_backends`